prowler/providers/gcp/exceptions/exceptions.py

```diff
@@ -37,6 +37,10 @@ class GCPBaseException(ProwlerException):
             "message": "Error loading static credentials",
             "remediation": "Check the credentials and ensure they are properly set up. client_id, client_secret and refresh_token are required.",
         },
+        (1933, "GCPNotValidProviderIdError"): {
```

Comment:
```suggestion
        (1933, "GCPInvalidAccountCredentials"): {
```

The point here is to validate that the provided credentials does not match with the `project_id` that wants to be used.

Reply:
Done!